packages/fluentui/react-northstar-emotion-renderer/src/createEmotionRenderer.tsx

```diff
@@ -30,7 +30,7 @@ export function createEmotionRenderer(options: CreateEmotionRendererOptions = {}
       container: target?.head,
       key: 'fui',
       nonce,
-      stylisPlugins: [focusVisiblePlugin],
+      stylisPlugins: [prefixer as StylisPlugin, focusVisiblePlugin as StylisPlugin],
```

Comment:
Emotion does not merge this setting, that's why we should include `prefixer` manually:

https://github.com/emotion-js/emotion/blob/3a8eaac14c1d157d0b5bb96597444e05f4c33eb1/packages/cache/src/index.js#L83

Reply:
nit: Let's store as our `defaultStylisPlugins` and reuse for the rtl case:
```suggestion
      stylisPlugins: defaultStylisPlugins,
```

Reply:
Done in a66c2b5.